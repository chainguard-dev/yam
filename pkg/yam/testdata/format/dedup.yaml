fruits:
  - apple
  - banana
  - apple
  - orange
  - banana
  - cherry
  - apple

vegetables:
  - carrot
  - potato
  - carrot
  - broccoli
  - potato

mixed:
  - item1
  - item2
  - item1
  - item3
  - item2

nested_map:
  nested:
    - sub1
    - sub2
    - sub1 