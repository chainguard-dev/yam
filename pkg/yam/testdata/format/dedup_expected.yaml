fruits:
  - apple
  - banana
  - orange
  - cherry
vegetables:
  - carrot
  - potato
  - broccoli
mixed:
  - item1
  - item2
  - item3
nested_map:
  nested:
    - sub1
    - sub2
